src/pages/Products/Productdetail/Productdetail.js

```diff
@@ -1,8 +1,264 @@
+            <img
+              className="sampleImage"
+              alt="product"
+              src="./images/sheabutter_cream.png"
+            />
+          </div>
+        </section>
+        {/* 오른쪽 */}
+        <section className="productRight">
+          <div className="sectionsWrapper">
+            {/* 섹션1 */}
+            <div className="sectionOne sections">
+              <div className="productName">
+                {name}
+                <div className="iconsWrapper">
+                  <img className="heart" alt="heart" src="./images/heart.png" />
+                  <img className="share" alt="share" src="./images/share.png" />
+                </div>
+              </div>
+              <div className="productPrice">
+                ₩{Math.floor(price).toLocaleString('ko-KR')}
+              </div>
+            </div>
+            {/* 섹션2 */}
+            <div className="sectionTwo sections">
+              <div className="coupon">
+                <span className="contentsTitle">쿠폰</span>
+                <button className="couponBtn">
+                  멤버십 상시 10% 할인(110할인)
+                  <img
+                    className="download"
+                    alt="download"
+                    src="./images/down-arrow.png"
+                  />
+                </button>
+              </div>
+              <div className="tags">
+                <span className="contentsTitle">관련태그</span>
+                <span className="tagsWrapper">
+                  <span className="tagEach">#비건</span>
+                  <span className="tagEach">#데일리템</span>
+                  <span className="tagEach">#생기맑음</span>
+                  <span className="tagEach">#촉촉</span>
+                </span>
+              </div>
+              <div className="shipping">
+                <span className="contentsTitle">배송비</span>
+                <span className="shippingFee">
+                  2,750원(20,000원 이상 결제시 무료배송)
+                </span>
+              </div>
+              <div className="review">
+                <span className="contentsTitle">리뷰</span>
+                <span className="starWrapper">
+                  <img className="star" src="./images/star.png" />
+                  <img className="star" src="./images/star.png" />
+                  <img className="star" src="./images/star.png" />
+                  <img className="star" src="./images/star.png" />
+                  <img className="star" src="./images/star.png" />
+                </span>
+                <span className="reviewUnit">(7건)</span>
+              </div>
+            </div>
+            {/* 섹션3 */}
+            {/* 셀렉트박스 */}
+            <div className="sectionThree sections">
+              <div>
+                <div className="optionWrapper">
+                  <select className="selectBox" onChange={this.select}>
+                    <option value="옵션선택">옵션을 선택해주세요</option>
+                    {this.state.type.map((item, index) => {
```

Comment:
```suggestion
                    {type.map((item, index) => {
```
여기도 구조분해 할당 해줄 수 있겠어요